blog/2024-01-12-pulsar-2023-year-in-review.md

```diff
@@ -0,0 +1,120 @@
+
+- **639** global contributors for the Pulsar main GitHub repository
+- **13.4K+** GitHub stars
+- **3.5K+** forks
+- **8** new Committers
+- **6** new PMC members
+- **10k+** Pulsar Slack members
+- **20M+** Docker pulls
+
+The Pulsar community welcomes all kinds of contributions. For more information, see the [Apache Pulsar Contribution Guide](pathname:///contribute/).
+
+## Project releases
+
+In 2023, the Pulsar community worked hard to improve the project’s capabilities and fix existing bugs with 2 major versions and 12 minor versions.
+
+The Apache Pulsar community released version [2.11](https://pulsar.apache.org/blog/2023/01/20/Apache-Pulsar-2-11-0/) with 61 contributors providing feature enhancements and fixes that delivered 1617 commits. 
+
+A Big Milestone for the Apache Pulsar community was the release of [Apache Pulsar 3.0](https://pulsar.apache.org/blog/2023/05/02/announcing-apache-pulsar-3-0/), the first Long-Term Support (LTS) version! The community is getting bigger! Over 140 contributors submitted about 1500 commits to the Pulsar 3.0 release, which is the largest contribution yet for a project that is fast becoming one of the biggest open-source projects. It includes support for LTS, which delivers the predictability and stability that larger enterprise teams need to deliver a solid and reliable messaging and streaming service.
+
+Thanks for all your contributions!
+
+Many important Pulsar capabilities were delivered in these releases, such as [Extensible Load Balancer](https://github.com/apache/pulsar/issues/16691) and [Large-scale delayed message support](https://github.com/apache/pulsar/issues/16763). For more information, see the [Release Notes page](pathname:///release-notes/).
+
+Updates about clients, Pulsar Manager, and Pulsar Helm Chart are listed below:
+
+- [Pulsar C++ Client 3.4.2](https://github.com/apache/pulsar-client-cpp/releases/tag/v3.4.2)
+- [Pulsar Go Client 0.11.1](https://github.com/apache/pulsar-client-go/releases/tag/v0.11.1)
+- [Pulsar Node.js Client 1.9.0](https://github.com/apache/pulsar-client-node/releases/tag/v1.9.0)
+- [Pulsar Python Client 3.3.0](https://github.com/apache/pulsar-client-python/releases/tag/v3.3.0)
+- [Pulsar Manager 0.4.0](https://github.com/apache/pulsar-manager/releases/tag/v0.4.0)
+- [Pulsar Helm Chart 3.1.0](https://github.com/apache/pulsar-helm-chart/releases/tag/pulsar-3.1.0)
+- [Pulsar dotnet Client 3.1.1](https://github.com/apache/pulsar-dotpulsar/blob/master/CHANGELOG.md#311---2023-12-11)
+- [Reactive Client for Apache Pulsar 0.1.0](https://github.com/apache/pulsar-client-reactive/releases/tag/v0.5.1)
+
+For more information, see the [Clients Release Notes page](pathname:///release-notes/clients/).
+
+# What’s next in 2024
+
+## Community Improvements
+The Pulsar community seeks to further improve the project from different aspects. For example, made the proposal to drop Pulsar SQL (Trino/Presto) in the 3.2.0 release. Start docker image vulnerability scans soon. 
```

Comment:
> to further improve the project from different aspects. For example, made the proposal to drop Pulsar SQL

Without additional context, it may be not clear how dropping something is considered an improvement.

Reply:
Maybe it makes sense to mention Oxia and metrics improvements?

Reply:
That makes sense. Thank you for that. I'm working on adding content on Pulsar SQL removal, Oxia & metrics

Reply:
@asafm added metrics improvements.